### PR TITLE
Added multiple frequency options

### PR DESCRIPTION
**Pull request by zen-mychen**
Enable the options to choose frequency of multiple hours, days, or weeks as described for "period" at
http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-object-schedule.html

Tested on our own pipeline

### CODE REVIEW

File: dataduct/pipeline/schedule.py [hunk 1]
```diff
@@ -19,8 +19,20 @@
 
 FEQUENCY_PERIOD_CONVERTION = {
     'weekly': ('1 week', None),
+    'biweekly': ('2 weeks', None),
```
Comment (tpcstld) [inline dataduct/pipeline/schedule.py]: Should call this 2-weekly to remain consistent

**Comment by sb2nov** [comment]
Actually I think `2-days` might be a better name than `2-daily` as that is more intuitive so may be we can come up with something like that. Thoughts ?

**Comment by tpcstld** [comment]
2-days sounds better than 2-daily. We should also add 1-day, 1-week, etc... as well to the map then as aliases.

Has it been considered to do everything in hours/minutes for the ultimate consistency? I.e. pass in 24 hours instead of 1 day, 168 hours instead of 1 week.

**Comment by zen-mychen** [comment]
Modified to include the nice comments.
Does everything in hours/minutes may bring a conversion every time people put the frequency?

**Comment by tpcstld** [comment]
I was wondering if we could just send "12345 minutes" in the API call and have AWS figure it out themselves. It's not important. (:

Anyways, LGTM. ~

**Comment by sb2nov** [comment]
Thanks for the change @zen-mychen

**Pull request merged**